### DISCUSSION TITLE
Creating a RPi to zigbee "bridge" using Sonoff  Zigbee 3.0 USB Dongle Plus, not sure if right

### DISCUSSION THREAD

**Discussion by lsd87** [Koenkk/zigbee2mqtt#11167] [General]
Hi - First a bit of info on what I'm trying to do,
I have a zigbee network running in my house (Homeassistant + ZHA + Conbee - may change ZHA to zigbee2mqtt later), now I'd like to add some zigbee sensors in a building thats over 100M away. (has wired Ethernet to building)
Last year I used a RPI3+ a CC2531 USB + zigbee2mqtt as a kind of bridge/router and this worked but range was not good - assumed cheap 2351. 

Now to the point 
Got a couple of Sonoff  Zigbee 3.0 USB Dongle Plus, installed latest RPi OS, and latest zigbee2mqtt. 
Tried the standard ITEAD supplied firmware and it works, I get messages in the MQTT server
So since (My understanding here), you only have one coordinator, I installed the lauchpad_router_20210128 and on running "npm start" - zigbee-herdsman reports "Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)"
If I reprogram the dongle to lauchpad_coordinator_20211217 and rerun npm start - zigbee2mqtt works (No changes to config files)
Tried various version of router from Zstack and get the same.

From the above the SONOFF's can be reprogrammed, and do work as coordinators 
SO -
1 Does zigbee2mqtt only work with coordinators ?
2 If so will it be a problem running 2 coordinators talking to the same mqtt server - both coordinators won't see each other over wifi
3 If the SONOFF has to be a router, any ideas ? 

Can add extra log info but would like to know if I'm going the right way :-)

Leigh

**Comment by francisp2** [comment]
>Does zigbee2mqtt only work with coordinators ?

Zigbee2mqtt needs a coordinator, you can add routers, but one (and only one) coordinator is required

>If so will it be a problem running 2 coordinators talking to the same mqtt server - both coordinators won't see each other over wifi
Install 2 instances of Zigbee2mqtt, and use a different mqtt topic and panid on each

**Comment by lsd87** [comment]
Thanks @francisp2, from your info and watching zigstar lan gateway video [https://www.youtube.com/watch?v=KBEaVZyvSgY](url) its looks like I need to install coordinator on the SONOFF stick / ruining on a RPi zigbee2mqtt (in the other building) and have it talk to my mqtt server. 
Then on the HA, run one zigbee2mqtt and talk to the same mqtt server as the RPI.
Will need to play with topics and panid to see what works
